docs/software_setup/sd_card.md

```diff
@@ -3,94 +3,86 @@
 
-1. Download the expandable JetBot SD card image [jetbot_image_v0p4p0.zip](https://drive.google.com/open?id=1G5nw0o3Q6E08xZM99ZfzQAe7-qAXxzHN)
+Download the pre-built JetBot SD card image from the table below.  Make sure to select the version that matches the Jetson you're using  (for example Jetson Nano 2GB).
 
-    ???+ info
-        The above image is based on JetPack 4.3.  The previous image based on JetPack 4.2 may be found here: [jetbot_image_v0p3p2.zip](https://drive.google.com/open?id=1GF2D814hkViwluZ5SgNKW56cQu_5Ekt5).
+### Latest Release
+
+| Platform | JetPack Version | JetBot Version | Download |
+|--------|-----------------|----------------|--------|
+| Jetson Nano 2GB | 4.4.1 | 0.4.1 | [jetbot-041_nano-2gb-jp441.zip](https://drive.google.com/file/d/1d03TOrQyffxFsv_Nhp-XQ7Q3-nCHbT9a/view) |
+| Jetson Nano (4GB) | 4.4.1 | 0.4.1 | [jetbot-041_nano-4gb-jp441.zip](https://drive.google.com/file/d/1yQ5MEiiBxbytCXHFPPBi-5SAxWklhZQA/view) |
+
+### Old releases
+
+| Platform | JetPack Version | JetBot Version | Download |
+|--------|-----------------|----------------|--------|
+| Jetson Nano (4GB) | 4.3 | 0.4.0 | [jetbot_image_v0p4p0.zip](https://drive.google.com/open?id=1G5nw0o3Q6E08xZM99ZfzQAe7-qAXxzHN) |
+| Jetson Nano (4GB) | 4.2 | 0.3.2 | [jetbot_image_v0p3p2.zip](https://drive.google.com/open?id=1GF2D814hkViwluZ5SgNKW56cQu_5Ekt5) | 
+
+## Step 2 - Flash JetBot image onto SD card
+
+1. Insert an SD card into your desktop machine
+
+3. Using [Etcher](https://www.balena.io/etcher/), select the image you downloaded above and flash it onto the SD card.
 
-2. Insert an SD card into your desktop machine
-3. Using [Etcher](https://www.balena.io/etcher/), select the ``jetbot_image_v0p4p0.zip`` image and flash it onto the SD card
 4. Remove the SD card from your desktop machine
 
-## Step 2 - Boot Jetson Nano
+## Step 3 - Boot Jetson Nano
 
 1. Insert the SD card into your Jetson Nano (the micro SD card slot is located 
    under the module)
 
 2. Connect the monitor, keyboard, and mouse to the Nano
-3. Power on the Jetson Nano by connecting the micro USB charger to the micro USB port
+3. Power on the Jetson Nano by connecting the micro USB (for Jetson Nano (4GB)) or USB-C (for Jetson Nano 2GB) charger to the port
 
     ???+ attention 
         We recommend first booting the Jetson Nano once without the piOLED / motor driver connected.
 
         This way you can check to make sure the system boots properly from the SD card image without worrying about hardware issues.  After you've verified that it boots, reconnect the piOLED, double check your wiring, and boot again.
 
-## Step 3 - Connect JetBot to WiFi
+## Step 4 - Connect JetBot to WiFi
 
-1. Log in using the user ``jetbot`` and password ``jetbot``
+Next you'll need to connect to WiFi.  To reduce memory consumption, we disable the Ubuntu GUI in the JetBot SD card image.  For this reason, you'll need to use the command line to connect to WiFi.
```

Comment:
Should we say "we disable the Ubuntu GUI in the **_latest_** JetBot SD card image."?

Reply:
Yes, good catch.  Added "latest"